DependencyInjection/CmfSeoExtension.php

```diff
@@ -197,21 +200,27 @@ private function loadAlternateLocaleProvider($config, ContainerBuilder $containe
             ? $this->defaultAlternateLocaleProviderId
             : $config['provider_id'];
 
+        if (!$alternateLocaleProvider) {
+            return;
+        }
 
-        if ($alternateLocaleProvider) {
-            $alternateLocaleProviderDefinition = $container->findDefinition($alternateLocaleProvider);
+        $alternateLocaleProviderDefinition = $container->findDefinition($alternateLocaleProvider);
+        if ($container->hasDefinition('cmf_seo.event_listener.seo_content')) {
             $container
                 ->findDefinition('cmf_seo.event_listener.seo_content')
                 ->addMethodCall(
                     'setAlternateLocaleProvider',
                     array($alternateLocaleProviderDefinition)
                 );
+        }
+        if ($container->hasDefinition('cmf_seo.sitemap.phpcr_provider')) {
```

Comment:
why try-catch? that would abort me.

hm, there is one reason to check $config: we detect when the service is expected to be loaded but not found (typo)